### ISSUE TITLE
Add doc.design.unload method

### ISSUE THREAD

**Issue by marcbachmann** [upfrontIO/livingdocs-engine#177]
We currently have a conflict in our html to livingdocs importer tests.
We initialized a new design with the same name like an already loaded design. (should this log something in the future?)
Beside the feature to load a specific design version, we probably need a design unload method.

**Comment by peyerluk** [comment]
> Should this log something in the future?

Yes, that sounds reasonable.

> Beside the feature to load a specific design version, we probably need a design unload method.

Do we really need that? For testing? I did not quite get the reason you hinted at.

**Comment by marcbachmann** [comment]
Yes mostly for testing. But also in node we should unload the designs after they aren't used for some time. Otherwise the memory usage would grow.

**Comment by marcbachmann** [comment]
But that's a wish for the far future. :smile:
We don't have memory problems the next year.

**Comment by peyerluk** [comment]
Ok. Then let's add this thing! :)

**Comment by marcbachmann** [comment]
Another possibility is to pass the design directly with the engine.createLivingdoc() method.
